### PR TITLE
fix: corrected automerge in renovate.json to true

### PR DESCRIPTION
**Pull request by Sigumaa**
fix: #95

### CODE REVIEW

**Review by re-taro — APPROVED** [review]

**Review by mirror-kt — CHANGES_REQUESTED** [review]
まだ見てないからマージ防止のために一旦RC

File: renovate.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "extends": ["config:base", ":timezone(Asia/Tokyo)"],
   "labels": ["dependencies", "renovate"],
   "pin": {
-    "automerge": false
+    "automerge": true
```
Comment (mirror-kt) [inline renovate.json:5]:
```suggestion
```

File: renovate.json [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
   },
   "packageRules": [
     {
       "groupName": "types",
       "packagePatterns": ["^@types/"],
-      "automerge": false,
+      "automerge": true,
```
Comment (mirror-kt) [inline renovate.json:11]:
```suggestion
```

File: renovate.json [hunk 3]
```diff
@@ -17,15 +17,15 @@
       "groupName": "linters",
       "extends": ["packages:linters"],
       "packagePatterns": ["^@typescript-eslint/"],
-      "automerge": false,
+      "automerge": true,
```
Comment (mirror-kt) [inline renovate.json:20]:
```suggestion
```

File: renovate.json [hunk 4]
```diff
@@ -17,15 +17,15 @@
...
       "major": {
         "automerge": false
       }
     },
     {
       "groupName": "storybook",
       "packagePatterns": ["^@storybook/"],
-      "automerge": false,
+      "automerge": true,
```
Comment (mirror-kt) [inline renovate.json:28]:
```suggestion
```

**Review by mirror-kt — CHANGES_REQUESTED** [review]
`packageRules` に以下を追加してください

```yaml
    {
      "matchUpdateTypes": [
        "minor",
        "patch"
      ],
      "automerge": true
    }
```

File: renovate.json (hunk 4 above)
Comment (Sigumaa) [reply renovate.json]: [27b4253](https://github.com/Growthers/enshita/pull/96/commits/27b425352f53edcfaac8a12c418dd5854ab3ae52)

File: renovate.json (hunk 3 above)
Comment (Sigumaa) [reply renovate.json]: [27b4253](https://github.com/Growthers/enshita/pull/96/commits/27b425352f53edcfaac8a12c418dd5854ab3ae52)

File: renovate.json (hunk 2 above)
Comment (Sigumaa) [reply renovate.json]: [27b4253](https://github.com/Growthers/enshita/pull/96/commits/27b425352f53edcfaac8a12c418dd5854ab3ae52)

File: renovate.json (hunk 1 above)
Comment (Sigumaa) [reply renovate.json]: [27b4253](https://github.com/Growthers/enshita/pull/96/commits/27b425352f53edcfaac8a12c418dd5854ab3ae52)

File: renovate.json [hunk 5]
```diff
@@ -1,34 +1,26 @@
...
-    "automerge": false
-  },
   "packageRules": [
+    {
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "automerge": true
+    }
```
Comment (mirror-kt) [inline renovate.json:11]:
```suggestion
    },
```

**Review by mirror-kt — CHANGES_REQUESTED** [review]
JSONとして不正です

File: renovate.json (hunk 5 above)
Comment (Sigumaa) [reply renovate.json]: [a8d752a](https://github.com/Growthers/enshita/pull/96/commits/a8d752a13701574ef96f7f3a6c5a5388d3eff8e5)

**Review by mirror-kt — APPROVED** [review]

**Review by re-taro — APPROVED** [review]

**Pull request merged**